### DISCUSSION TITLE
Concatenate layer support in Loihi

### DISCUSSION THREAD

**Discussion by lsy105** [lava-nc/lava-dl#183] [Q&A] [answered]
Hello everyone, 

I was wondering if it is possible to implement a concatenate layer for Loihi using lava-dl. The following link (https://lava-nc.org/lava-lib-dl/netx/netx.html) says concatenate layer is supported in Loihi. However, I could not find the concatenate layer in the lava-dl documentation. I was wondering if there is an example code about how to implement a new layer for Loihi using lava-dl.

Thank you!

**Comment by bamsumit** [comment, marked as answer]
Hi @lsy105, thanks for reaching out. This feature has not been implemented in NetX yet. The reason is that concatenation of ports is not supported in Lava (see [here](https://github.com/lava-nc/lava/blob/main/src/lava/magma/core/process/ports/ports.py#L219)). Once it is enabled, it is easy to add support on NetX export and network generation.

I would recommend requesting support for __concatenating ports__ on [Lava issues](https://github.com/lava-nc/lava/issues) to push the development of this feature up in the Lava dev team's priority.
